src/internal/colors/RgbwwColor.cpp

```diff
@@ -0,0 +1,217 @@
+/*-------------------------------------------------------------------------
+RgbwColor provides a color object that can be directly consumed by NeoPixelBus
+
+Written by Michael C. Miller.
+
+I invest time and resources providing this open source code,
+please support me by dontating (see https://github.com/Makuna/NeoPixelBus)
+
+-------------------------------------------------------------------------
+This file is part of the Makuna/NeoPixelBus library.
+
+NeoPixelBus is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
+
+NeoPixelBus is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with NeoPixel.  If not, see
+<http://www.gnu.org/licenses/>.
+-------------------------------------------------------------------------*/
+
+#include <Arduino.h>
+#include "../NeoSettings.h"
+#include "RgbColorBase.h"
+#include "RgbColor.h"
+#include "Rgb48Color.h"
+#include "HslColor.h"
+#include "HsbColor.h"
+#include "RgbwwColor.h"
+#include "HtmlColor.h"
+
+RgbwwColor::RgbwwColor(const HtmlColor& color)
+{
+    uint32_t temp = color.Color;
+    B = (temp & 0xff);
+    temp = temp >> 8;
+    G = (temp & 0xff);
+    temp = temp >> 8;
+    R = (temp & 0xff);
+    temp = temp >> 8;
+    WW = (temp & 0xff);
+    temp = temp >> 8;
+    CW = (temp & 0xff);
```

Comment:
CW will always be zero (32 bits already shifted right 4 times).
Might consider setting WW and CW the same or at 50% of the fourth byte.
Not the biggest deal as we are already abusing a fourth byte of an html color which isn't well documented.